.buildkite/bk.integration-fips.pipeline.yml

```diff
@@ -0,0 +1,145 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
+
+env:
+  DOCKER_REGISTRY: "docker.elastic.co"
```

Comment:
Do we know whether we need this env variable? I cannot find any references https://github.com/search?q=repo%3Aelastic%2Felastic-agent%20DOCKER_REGISTRY&type=code?

Reply:
I think this is needed by the docker login

Reply:
I'll keep a note here to revisit this, thanks Michel!